### PR TITLE
Support issue template assignees

### LINKED ISSUE #13955: Add assignees support to Issue template
- Gitea version (or commit ref): 1.13.0
- Git version: n/a
- Operating system: n/a
  
  
  
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No
- Log gist:

## Description
Add support for assignees in ISSUE_TEMPLATE.md in order to add a user directly when creating an issue from a template. This is supported in GitHub ([see docs](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/manually-creating-a-single-issue-template-for-your-repository)).

Example of usage: My colleague is also responsible for a server of the product. Issues related to the server will be always assigned to him. This could ease the process of creation of the issue.

Proposed format:

```
---

name: "Template Name"
about: "This template is for testing!"
title: "[TEST] "
labels:

- bug
- "help needed"

assignees:

- user1

---

This is the template!
```

### PR DESCRIPTION
**Pull request by Zettat123**
Resolve #13955

### CODE REVIEW

**Comment by lunny** [comment]
Please also update the documentations

File: routers/web/repo/issue.go [hunk 1]
```diff
@@ -934,12 +934,23 @@ func setTemplateIfExists(ctx *context.Context, ctxDataKey string, possibleFiles
...
+		}
 
 		if template.Ref != "" && !strings.HasPrefix(template.Ref, "refs/") { // Assume that the ref intended is always a branch - for tags users should use refs/tags/<ref>
 			template.Ref = git.BranchPrefix + template.Ref
 		}
 		ctx.Data["HasSelectedLabel"] = len(labelIDs) > 0
 		ctx.Data["label_ids"] = strings.Join(labelIDs, ",")
+		ctx.Data["HasSelectedAssignee"] = len(selectedAssigneeIDs) > 0
+		ctx.Data["selected_assignee_ids_value"] = strings.Join(selectedAssigneeIDStrings, ",")
+		ctx.Data["selected_assignee_ids"] = selectedAssigneeIDs
```
Comment (delvh) [inline routers/web/repo/issue.go]: Why are we mixing camelCase and snake_case in the same method?
Can we simply keep using CamelCase?

**Comment by lunny** [comment]
Please move docs to https://gitea.com/gitea/docs. Sorry for the inconvinence.

**Comment by Zettat123** [comment]
> Please move docs to https://gitea.com/gitea/docs. Sorry for the inconvinence.

Created a [PR](https://gitea.com/gitea/docs/pulls/48) for docs, please help review

**Review by wolfogre — APPROVED** [review]

File: templates/repo/issue/new_form.tmpl [hunk 2]
```diff
@@ -140,7 +140,7 @@
 		</div>
 		{{end}}
 		<div class="divider"></div>
-			<input id="assignee_ids" name="assignee_ids" type="hidden" value="{{.assignee_ids}}">
```
Comment (lunny) [inline templates/repo/issue/new_form.tmpl]: I think `.assignee_ids` has been used in repo_form.go:445? Can we ignore it directly?

**Review by lunny — APPROVED** [review]

File: templates/repo/issue/new_form.tmpl (hunk 2 above)
Comment (Zettat123) [reply templates/repo/issue/new_form.tmpl]: I renamed this variable and continued using `assignee_ids`. See de17b5ee5b35491d860b380b9f3cef2fe77fb312

**Pull request merged**